crates/krun/src/guest/mount.rs

```diff
@@ -80,11 +80,13 @@ fn mount_fex_rootfs() -> Result<()> {
     // and a Config.json telling FEX to use FUSE. Neither should be visible to the guest. Instead,
     // we want to replace the folders and tell FEX to use our mounted rootfs
     for base in ["/usr/share/fex-emu", "/usr/local/share/fex-emu"] {
-        let json = format!("{{\"Config\":{{\"RootFS\":\"{dir_rootfs}\"}}}}\n");
-        let path = base.to_string() + "/Config.json";
+        if Path::new(base).exists() {
```

Comment:
```suggestion
        if fs::exists(base) {
```

https://doc.rust-lang.org/stable/std/fs/fn.exists.html

Reply:
the doc mentioned TOCTOU bugs and then I realized there's a better way. fixed.